lib/vagrant/util/ipv4_interfaces.rb

```diff
@@ -0,0 +1,13 @@
+module Vagrant
+  module Util
+    module IPv4Interfaces
+      def self.ipv4_interfaces
+        Socket.getifaddrs.select do |ifaddr|
+          ifaddr.addr && ifaddr.addr.ipv4?
+        end.map do |ifaddr|
+          [ifaddr.name, ifaddr.addr.ip_address]
+        end
+      end
```

Comment:
And then add an `extend IPv4Interfaces` so you can still call it like you are currently.